grammars/javascript.cson

```diff
@@ -1251,3 +1322,82 @@
         ]
       }
     ]
+  'switch_statement':
+    'patterns': [
+      {
+        'begin': '\\b(switch)\\b'
```

Comment:
Also, what do you say about assigning the entire switch statement a scope like `meta.switch-statement.js`, and the body of it `meta.switch-statement.body.js` (same thing with cases)?  This brings us in line with other languages like C and Java.  As an added benefit it'll help autocompletion.